workspace_api/views.py

```diff
@@ -97,7 +98,31 @@ async def create_workspace(
         )
     )
 
-    create_bucket(workspace_name=workspace_name)
+    group = "epca.eo"
+    version = "v1alpha1"
+    body = {
+        "apiVersion": f"{group}/{version}",
+        "kind": "Bucket",
+        "metadata": {
+            # TODO: better name for bucket resource?
+            "name": workspace_name,
+            "namespace": config.NAMESPACE_FOR_BUCKET_RESOURCE,
+        },
+        "spec": {
+            # we use the workspace name as bucket name since it's a good unique name
+            "bucketName": workspace_name,
+            "secretName": config.WORKSPACE_SECRET_NAME,
+            "secretNamespace": workspace_name,
+        },
+    }
```

Comment:
I wouldn't put the whole kubernetes CRD definition in the request body, this is unnecessary. I'd just use the `spec`:
```python
body = {
     # we use the workspace name as bucket name since it's a good unique name
    "bucketName": workspace_name,
    "secretName": config.WORKSPACE_SECRET_NAME,
    "secretNamespace": workspace_name,
}
```

Also: `version` and `group` are not used